distributed/cli/dask_scheduler.py

```diff
@@ -110,6 +122,8 @@ def del_pid_file():
         scheduler.stop()
         if bokeh_proc:
             bokeh_proc.close()
+        if os.path.exists(local_directory):
```

Comment:
This is the way `Worker` handles its local directory, I just did the same thing.